addons/product/views/product_views.xml

```diff
@@ -24,7 +24,12 @@
                     <field name="image_1920" widget="image" class="oe_avatar" options="{'preview_image': 'image_128'}"/>
                     <div class="oe_title">
                         <label for="name" string="Product Name"/>
-                        <h1><field class="o_text_overflow" name="name" placeholder="e.g. Cheese Burger"/></h1>
+                        <h1>
+                            <div style="display: flex">
+                                <field name="priority" widget="priority" class="mr-3"/>
+                                <field class="o_text_overflow" name="name" placeholder="e.g. Cheese Burger"/>
+                            </div>
+                        </h1>
```

Comment:
```suggestion
                        <h1 class="d-flex">
                            <field name="priority" widget="priority" class="mr-3"/>
                            <field class="o_text_overflow" name="name" placeholder="e.g. Cheese Burger"/>
                        </h1>
```

Also, you can edit your commit's message to remove the `Improve autocomplete of products in customer/vendor documents` as this part was removed.